lib/types/duration.js

```diff
@@ -102,15 +96,15 @@ Duration.prototype.toString = function () {
     value += dividend.divide(divisor).toString() + unit;
     return dividend.modulo(divisor);
   }
-  if (this.months < 0 || this.days < 0 || this.nanoseconds.isNegative()) {
+  if (this.internal.months < 0 || this.internal.days < 0 || nanoseconds.isNegative()) {
```

Comment:
why did `this.nanoseconds` change into `nanoseconds`? Are both equivalent in JS?

Reply:
this.internal.nanoseconds refers to the value in rust object. In JS it's of bigint type. This datastax code treats nanoseconds as Long, so we need to convert the type, which we keep in local variable nanoseconds.

Reply:
My bad, I didn't notice the variable. Nvm.